torchdynamo/variables/torch.py

```diff
@@ -244,6 +246,79 @@ def fake_softmax(input):
 
         return variables.LambdaVariable(fake_softmax, **options)
 
+    def _call_cross_entropy_loss(self, tx, args, kwargs, options):
+        """
+                functional: input, target, weight=None, size_average=None, ignore_index=- 100, reduce=None, reduction='mean', label_smoothing=0.0
+
+        non functional ctor: weight=None, size_average=None, ignore_index=- 100, reduce=None, reduction='mean', label_smoothing=0.0
+
+        non functional loss call: input, target, optional_output
+        """
+        # TODO(voz): This feels tantalizingly close to something we could just codegen?
+        weight = (
+            args[0] if args else kwargs.get("weight", variables.ConstantVariable(None))
+        )
+        size_average = (
+            args[1]
+            if args
+            else kwargs.get("size_average", variables.ConstantVariable(None))
+        )
+        ignore_index = (
+            args[2]
+            if args
+            else kwargs.get("ignore_index", variables.ConstantVariable(None))
+        )
+        reduce_arg = (
+            args[3] if args else kwargs.get("reduce", variables.ConstantVariable(None))
+        )
+        reduction = (
+            args[4]
+            if args
+            else kwargs.get("reduction", variables.ConstantVariable(None))
+        )
+        label_smoothing = (
+            args[5]
```

Comment:
This will throw an exception if you only give one arg.  I'd suggest:

```
def normalize_args(weight=ConstantVariable(None), size_average=ConstantVariable(None), ignore_index=ConstantVariable(-100), reduce=ConstantVariable(None), reduction=ConstantVariable('mean'), label_smoothing=ConstantVariable(0.0)):
   return weight, size_average, ignore_index, reduce, reduction, label_smoothing
   
weight, size_average, ignore_index, reduce, reduction, label_smoothing = normalize_args(*args, **kwargs) 
```